src/main/java/edu/iis/powp/command/CommandSetPosition.java

```diff
@@ -0,0 +1,16 @@
+package edu.iis.powp.command;
+
+import edu.iis.client.plottermagic.IPlotter;
+
+public class CommandSetPosition implements PlotterCommand {
+	
+	private int x;
+	private int y;
+	
+	@Override
+	public void execute(IPlotter iPlotter) {
```

Comment:
iPlotter is not a good name for a parameter (plotter would be better). 'I' prefix is used in types to indicate an interface.